neptune/internal/storage/storage_utils.py

```diff
@@ -134,7 +134,7 @@ def split_upload_files(upload_entries, max_package_size=1 * 1024 * 1024, max_fil
 
 
 def normalize_file_name(name):
-    return name.replace(os.sep, '/')
+    return name.replace(sep, '/')
```

Comment:
I think this code is confusing long-term - sep looks like a normal variable and I need to look up imports to understand what it is